<!DOCTYPE html>
<html>
<head>
  <title>Gene Expression &amp; Growth &lt;br/&gt;in Budding Yeast</title>
  <meta charset="utf-8">
  <meta name="description" content="Gene Expression &amp; Growth &lt;br/&gt;in Budding Yeast">
  <meta name="author" content="Rainer Machne">
  <meta name="generator" content="slidify" />
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta http-equiv="X-UA-Compatible" content="chrome=1">
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/default.css" media="all" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/phone.css" 
    media="only screen and (max-device-width: 480px)" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/slidify.css" >
  <link rel="stylesheet" href="libraries/highlighters/highlight.js/css/tomorrow.css" />
  <base target="_blank"> <!-- This amazingness opens all links in a new tab. -->  <link rel=stylesheet href="./assets/css/my.css"></link>
<link rel=stylesheet href="./assets/css/ribbons.css"></link>

  
  <!-- Grab CDN jQuery, fall back to local if offline -->
  <script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>
  <script>window.jQuery || document.write('<script src="libraries/widgets/quiz/js/jquery.js"><\/script>')</script> 
  <script data-main="libraries/frameworks/io2012/js/slides" 
    src="libraries/frameworks/io2012/js/require-1.0.8.min.js">
  </script>
  
  

</head>
<body style="opacity: 0">
  <slides class="layout-widescreen">
    
    <!-- LOGO SLIDE -->
        <slide class="title-slide segue nobackground">
  <hgroup class="auto-fadein">
    <h1>Gene Expression &amp; Growth &lt;br/&gt;in Budding Yeast</h1>
    <h2>X-Men Group Retreat, Bergen aan Zee, Nov 2016</h2>
    <p>Rainer Machne<br/></p>
  </hgroup>
  <article></article>  
</slide>
    

    <!-- SLIDES -->
    <slide class="" id="slide-1" style="background:;">
  <article data-timings="">
    <style>
em {
  font-style: italic
}
</style>

<h2>Respiratory Oscillations in Budding Yeast</h2>

<table><thead>
<tr>
<th>ultradian</th>
<th>circadian</th>
</tr>
</thead><tbody>
<tr>
<td><em>Saccharomyces cerevisiae</em></td>
<td><em>Synechocystis</em> PCC6803</td>
</tr>
<tr>
<td><img src="assets/img/tuliCoarse_li06_timecourse_ranges_major_nrm.png" height="180"></td>
<td><img src="assets/img/cls_timecourse_major_total_fromLehmann14.png" height="190"></td>
</tr>
<tr>
<td><em>E. coli</em>  &amp; Paul, the <em>plateypus</em></td>
<td>RNA-seq &amp; Karl, the <em>segmenTier</em></td>
</tr>
<tr>
<td><img src="assets/img/ecoli_20141014.png" height="160"><img src="assets/img/fritz_the_platypus.gif" height="160"></td>
<td><img src="assets/img/anadenobolus_arboreus-90.png" height="160"></td>
</tr>
</tbody></table>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="class" id="id" style="background:;">
  <hgroup>
    <h3>Growth &amp; Gene Expression in <em>E. coli</em></h3>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/ecoli_20141014.png" height="250"><img src="assets/img/fritz_the_platypus.gif" height="230"><img src="assets/img/Ecoli_20161014_OD_grofit_A8.png" height="250"></p>

<p>\[ 
\begin{equation*} \begin{aligned}  
\frac{\text{d}X(t)}{\text{d}t} =& \mu X(t)\\ 
X(t) =& X(0)   e^{\mu  t}\\ 
t_D =& \frac{\ln(2)}{\mu}  
\end{aligned} \end{equation*} \]</p>

<p><br/>
<a href="https://github.com/raim/platexpress">https://github.com/raim/platexpress</a> or Paul, the <em>plateypus</em>?</p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="class" id="id" style="background:;">
  <hgroup>
    <h3>Growth &amp; Gene Expression in <em>E. coli</em></h3>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/ecoli_20141014.png" height="250"><img src="assets/img/fritz_the_platypus.gif" height="230"><img src="assets/img/Ecoli_20161014_OD_growthrates.png" height="250"></p>

<p>\[ 
\begin{equation*} \begin{aligned}  
\frac{\text{d}X(t)}{\text{d}t} =& \mu X(t)\\ 
X(t) =& X(0)   e^{\mu  t}\\ 
t_D =& \frac{\ln(2)}{\mu}  
\end{aligned} \end{equation*} \]</p>

<p><br/>
<a href="https://github.com/raim/platexpress">https://github.com/raim/platexpress</a> or Paul, the <em>plateypus</em>?</p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight" id="slide-4" style="background:;">
  <hgroup>
    <h4>Varying Periods</h4>
  </hgroup>
  <article data-timings="">
    <table><thead>
<tr>
<th>continuous culture</th>
<th>observed periods</th>
</tr>
</thead><tbody>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/fermentor_detailed.png" height="250"></td>
<td><img src="assets/img/strains_D_t.png" height="250"></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><font size=4>in continuous culture: \[ \begin{equation*} \begin{aligned} \frac{\text{d}X(t)}{\text{d}t} =& \mu X(t)- \phi X(t) \end{aligned} \end{equation*} \] </font><br/></td>
<td><font size=4>at steady state: \[\mu = \phi\] </font></td>
</tr>
</tbody></table>

<p><br/><br/>
<font size=3>Monod, Annual Review of Microbiology 1949: <em>The Growth of Bacterial Cultures.</em><br/>
Finn &amp; Wilson, J Agric Food Chem 1954: <em>Population Dynamics of a Continuous Propagator for Microorganisms.</em></font></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight" id="slide-5" style="background:;">
  <hgroup>
    <h4>Varying Periods at Balanced Growth</h4>
  </hgroup>
  <article data-timings="">
    <table><thead>
<tr>
<th>continuous culture</th>
<th>observed periods</th>
</tr>
</thead><tbody>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/fermentor_detailed.png" height="250"></td>
<td><img src="assets/img/strains_D_t.png" height="250"></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td>\[ \begin{equation*} \begin{aligned} \frac{\text{d}X(t)}{\text{d}t} =& \mu X(t)- \phi X(t) \\ \frac{\text{d}S(t)}{\text{d}t} =&  \phi (S_{in} - S(t)) - \frac{\mu}{Y} X(t) \end{aligned} \end{equation*} \]</td>
<td>\[\mu = \mu_{max}\frac{S^n}{S^n+K^n}\] \[Y = \frac{\Delta X}{\Delta S}  \]</td>
</tr>
</tbody></table>

<p><br/><br/>
<font size=3>Monod, Ann Rev Microbiol 1949: <em>The Growth of Bacterial Cultures.</em><br/>
Finn &amp; Wilson, J Agric Food Chem 1954: <em>Population Dynamics of a Continuous Propagator for Microorganisms.</em><br/>
Mochan &amp; Pye, Nature New Biol 1973: <em>Respiratory oscillations in adapting yeast cultures.</em><br/>
Heinzle <em>et al.</em>, 1983: <em>Modelling of sustained oscillations observed in continuous culture of Saccharomyces cerevisiae.</em>
</font></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight" id="slide-6" style="background:;">
  <hgroup>
    <h4>Varying Periods &amp; the Cell Cycle?</h4>
  </hgroup>
  <article data-timings="">
    <table><thead>
<tr>
<th>the budding cycle</th>
<th>observed periods</th>
</tr>
</thead><tbody>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><br/><img src="assets/img/thompson80_fig1_t2.png" height="220"></td>
<td><img src="assets/img/zoom_T2_t.png" height="250"></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/budding.svg" height="180"></td>
<td>\[\begin{equation*} \begin{aligned}t_D &= \frac{\ln(2)}{\mu} \\ \frac{t_O}{2} = t_P &= \frac{\ln\frac{1+\sqrt{5}}{2}}{\mu} \end{aligned} \end{equation*}\]</td>
</tr>
</tbody></table>

<p><font size=3>
Thompson &amp; Wheals, J Gen Microbiol 1980: <em>Asymmetrical division of Saccharomyces cerevisiae in glucose-limited chemostat culture.</em><br/>
Duboc, Philippe &amp; von Stockar, Chem Eng Sci 2000: <em>Modeling of oscillating cultivations of Saccharomyces cerevisiae</em> [...].<br/>
Klevecz <em>et al.</em>, PNAS 2004: <em>A genomewide oscillation in transcription gates DNA replication and cell cycle.</em>
</font></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="class" id="id" style="background:;">
  <hgroup>
    <h4>Varying Periods ... Constant Gene Expression Profile</h4>
  </hgroup>
  <article data-timings="">
    <table><thead>
<tr>
<th>Observed periods</th>
<th></th>
<th></th>
<th></th>
</tr>
</thead><tbody>
<tr>
<td><img src="assets/img/wang15_highD_ranges_major.png" height="120"></td>
<td>CEN.PK 122<br/>\(\phi=0.13\;\text{h}^{-1}\)<br/>\(t_D=5.3\;\text{h}\)<br/>\(t_O=2\;\text{h}\)</td>
<td><img src="assets/img/tuliCoarse_tu05_timecourse_ranges_major_nrm.png" height="120"></td>
<td>CEN.PK 122<br/>\(0.1\;\text{h}^{-1}\) <br/>\(t_D=6.9\;\text{h}\) <br/>\(t_O=5\;\text{h}\)</td>
</tr>
<tr>
<td><img src="assets/img/tuliCoarse_li06_timecourse_ranges_major_nrm.png" height="120"></td>
<td>IFO 0233<br/>\(0.09\;\text{h}^{-1}\)<br/>\(t_D=7.7\;\text{h}\) <br/>\(t_O=0.7\;\text{h}\)</td>
<td><img src="assets/img/wang15_lowD_ranges_major.png" height="120"></td>
<td>CEN.PK 122<br/>\(0.05\;\text{h}^{-1}\)<br/>\(t_D=13.9\;\text{h}\) <br/>\(t_O=7\;\text{h}\)</td>
</tr>
<tr>
<td><br/><div style='text-align: right;'><font size=5>Oscillation without<br/>(much) cell division:</font></div></td>
<td></td>
<td><img src="assets/img/slavov11_ranges_major.png" height="120"></td>
<td>DBY12007<br/>\(\sim \textbf{0}\;\text{h}^{-1}\) <br/>\(t_D=\inf \;\text{h}\) <br/>\(t_O=4\;\text{h}\)</td>
</tr>
</tbody></table>

<p><br/></p>

<div style='text-align: right;'>$$\rightarrow \text{live at the brink}$$</div>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight centertext" id="slide-8" style="background:;">
  <hgroup>
    <h3>Gene Expression : function</h3>
  </hgroup>
  <article data-timings="">
    <p><br/></p>

<p><img src="assets/img/clusterDAG_v2_nona.png" height="280"></p>

<p><br/>
based on <em>gene ontology</em> analysis</p>

<p><br/><br/>
<font size=3></p>

<div style='text-align: left;'>Murray <em>et al.</em>, PNAS 2007: <em>Regulation of yeast oscillatory dynamics.</em><br/>
Machne & Murray, PLoS ONE 2012: <em>The yin and yang of yeast transcription:<br/> elements of a global feedback system between metabolism and chromatin.</em>
</div>

<p></font></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight centertext" id="slide-9" style="background:;">
  <hgroup>
    <h3>Gene Expression : meta-transcriptome</h3>
  </hgroup>
  <article data-timings="">
    <p><br/></p>

<p><img src="assets/img/machne12_fig2_cropped.png" height="280"></p>

<p><br/>
cluster gene expression in ~1300 microarray experiments</p>

<p><br/><br/>
<font size=3></p>

<div style='text-align: left;'>Murray <em>et al.</em>, PNAS 2007: <em>Regulation of yeast oscillatory dynamics.</em><br/>
Machne & Murray, PLoS ONE 2012: <em>The yin and yang of yeast transcription:<br/> elements of a global feedback system between metabolism and chromatin.</em>
</div>

<p></font></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight centertext" id="slide-10" style="background:;">
  <hgroup>
    <h3>Gene Expression &amp; Growth Laws</h3>
  </hgroup>
  <article data-timings="">
    <p><br/></p>

<p><img src="assets/img/cluster_vs_rates_major.png" height="280"></p>

<p><br/>
\(\mu \sim \frac{\text{red genes}}{\text{blue genes}}\)</p>

<p><br/></p>

<div  style='text-align: left;'><font size=3> 
Brauer <em>et al.</em>, Mol Biol Cell 2008: <em>Coordination of growth rate, cell cycle, stress response, and metabolic activity in yeast.</em><br/>
Slavov <em>et al.</em>, Mol Biol Cell 2011: <em>Coupling among growth rate response, metabolic cycle, and cell division cycle in yeast.</em><br/>
</font></div>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight centertext" id="slide-11" style="background:;">
  <hgroup>
    <h3>Gene Expression &amp; Growth Laws</h3>
  </hgroup>
  <article data-timings="">
    <p><br/></p>

<p><img src="assets/img/cluster_vs_rates_major.png" height="280"></p>

<p><br/>
\(\mu = k \frac{\text{ribosomes}}{\text{proteins}}\)</p>

<p><br/></p>

<div  style='text-align: left;'><font size=3> 
Brauer <em>et al.</em>, Mol Biol Cell 2008: <em>Coordination of growth rate, cell cycle, stress response, and metabolic activity in yeast.</em><br/>
Slavov <em>et al.</em>, Mol Biol Cell 2011: <em>Coupling among growth rate response, metabolic cycle, and cell division cycle in yeast.</em><br/>
<b>Koch, Can J Microbiol 1988: <em>Why can't a cell grow infinitely fast?</em></b><br/>
</font></div>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="centertext lineheight" id="slide-12" style="background:;">
  <hgroup>
    <h3>Gene Expression &amp; Growth Laws</h3>
  </hgroup>
  <article data-timings="">
    <p><br/></p>

<p><img src="assets/img/cluster_vs_doubling_AD.png" height="280"></p>

<p><br/>
\(t_D=\frac{\ln(2)}{\mu}\)</p>

<p><br/></p>

<div  style='text-align: left; line-height: 90%;'><font size=3> 
Slavov <em>et al.</em>, Mol Biol Cell 2011: <em>Coupling among growth rate response, metabolic cycle, and cell division cycle in yeast.</em><br/>
Burnetti, Aydin & Buchler, Mol Biol Cell 2016: <em>Cell cycle Start is coupled to entry into the yeast metabolic cycle<br/>across diverse strains and growth rates.</em>
</font></div>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight" id="slide-13" style="background:;">
  <article data-timings="">
    <table><thead>
<tr>
<th>major carbon flux</th>
<th>nucleic acid content</th>
<th>cell</th>
</tr>
</thead><tbody>
<tr>
<td><img src="assets/img/data_DW_interpolated_nc.png"  height="180"></td>
<td><img src="assets/img/data_RNA_interpolated_nc.png"  height="180"></td>
<td><img src="assets/img/thompson80_fig1.png"  height="180"></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/data_EtOH_interpolated_nc.png"  height="180"></td>
<td><img src="assets/img/data_DNA_interpolated_nc.png"  height="180"></td>
<td><img src="assets/img/data_V_cell_interpolated_nc.png"  height="180"></td>
</tr>
</tbody></table>

<p><br/>
With increasing growth rate <em>&mu;</em>:<br/> 
higher RNA &amp; lower DNA content; larger cells, lower biomass &amp; higher ethanol yield!</p>

<p><br/><br/>
<font size=3> 
<b>Koch, Can J Microbiol 1988: <em>Why can&#39;t a cell grow infinitely fast?</em></b><br/>
Schaechter, Maaloe &amp; Kjeldgaard, J Gen Microbiol 1958: <em>Dependency on medium and temperature of cell size and chemical composition during balanced growth of Salmonella typhimurium.</em><br/>
Heijnen &amp; Van Dijken, Biotechnol Bioeng 1992: <em>In search of a thermodynamic description of biomass yields</em> [...].<br/>
Wei&szlig;e <em>et al.</em>, PNAS 1992: <em>Mechanistic links between cellular trade-offs, gene expression, and growth.</em>
</font></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight" id="slide-14" style="background:;">
  <article data-timings="">
    
<div style='float:left;width:51%;'>
  <h4>Summary:<br/>Growth &amp; Gene Expression</h4>

<p><br/></p>

<ol>
<li>Growth Rate vs. Protein Expression Trade-off</li>
<li>Respiratory Oscillation:<br/>varying periods != cell division cycle</li>
<li>Anti-correlation: Red \(\leftrightarrow\) Blue Genes</li>
<li>Relative Red/Blue Gene Expression<br/>
correlates with growth rate <em>&mu;</em><br/>
and with major cellular restructuring</li>
</ol>

<p><img src="assets/img/basic_feedback.png"  height="140"></p>

</div>
<div style='float:right;width:48%;'>
  <h4>A Working Hypothesis: Life&#39;s Fundamental Feedbacks</h4>

<p><br/><br/></p>

<p><font size=3>
\[   \begin{equation*}
    \label{eqn:ancat}
    \begin{aligned}
      \frac{\text{d}X}{\text{d}t} &= (\mu_{ab} - \phi) X  \\
      \frac{\text{d}S}{\text{d}t} &= \phi (S_{in} - S)-(\mu_{ab} + \mu_{cd}) X\\  
      \frac{\text{d}atp}{\text{d}t} &= (n_{cd} \mu_{cd} - n_{ab} \mu_{ab} - \mu_{m})\frac{C_c}{V_c} - \mu_{ab} atp\\
      adp & = a_{tot} - atp\\\hline
     \end{aligned}
  \end{equation*} \]
</font>
<br/><br/></p>

<p><img src="assets/img/model_feedback.png"  height="180"></p>

</div>
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight" id="slide-15" style="background:;">
  <article data-timings="">
    <table><thead>
<tr>
<th>diauxic growth</th>
<th>metabolism</th>
</tr>
</thead><tbody>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/zampar13_growth.png"  height="240"></td>
<td><img src="assets/img/corecarbon.svg"  height="240"></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/fermentor_detailed.png" height="240"></td>
<td></td>
</tr>
</tbody></table>

<p><font size=3> 
Zampar <em>et al.</em>, Mol Syst Biol 2013: <em>Temporal system-level organization of the switch from glycolytic to gluconeogenic operation in yeast.</em><br/>
</font></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight" id="slide-16" style="background:;">
  <article data-timings="">
    <table><thead>
<tr>
<th>diauxic growth</th>
<th>cell cycle</th>
</tr>
</thead><tbody>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/zampar13_growth.png"  height="240"></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/brauer05_cells.png"  height="240"></td>
<td></td>
</tr>
</tbody></table>

<p><font size=3> 
Zampar <em>et al.</em>, Mol Syst Biol 2013: <em>Temporal system-level organization of the switch from glycolytic to gluconeogenic operation in yeast.</em><br/>
Brauer <em>et al.</em>, Mol Biol Cell 2005: <em>Homeostatic adjustment and metabolic remodeling in glucose-limited yeast cultures.</em>
</font></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight" id="slide-17" style="background:;">
  <article data-timings="">
    <table><thead>
<tr>
<th>diauxic growth</th>
<th>metabolism</th>
</tr>
</thead><tbody>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/zampar13_growth.png"  height="240"></td>
<td><img src="assets/img/brauer05_cells.png"  height="240"></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/slavov14_growth.png"  height="240"></td>
<td></td>
</tr>
</tbody></table>

<p><font size=3> 
Zampar <em>et al.</em>, Mol Syst Biol 2013: <em>Temporal system-level organization of the switch from glycolytic to gluconeogenic operation in yeast.</em><br/>
Brauer <em>et al.</em>, Mol Biol Cell 2005: <em>Homeostatic adjustment and metabolic remodeling in glucose-limited yeast cultures.</em><br/>
Slavov <em>et al.</em>, Cell Rep 2014: <em>Constant growth rate can be supported by decreasing energy flux and increasing aerobic glycolysis.</em>
</font></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight" id="slide-18" style="background:;">
  <article data-timings="">
    <table><thead>
<tr>
<th>diauxic growth</th>
<th>transcription</th>
</tr>
</thead><tbody>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/zampar13_growth.png"  height="240"></td>
<td><img src="assets/img/brauer05_cells.png"  height="240"></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/slavov14_growth.png"  height="240"></td>
<td><img src="assets/img/slavov14_selected_transcripts.png"  height="240"></td>
</tr>
</tbody></table>

<p><font size=3> 
Zampar <em>et al.</em>, Mol Syst Biol 2013: <em>Temporal system-level organization of the switch from glycolytic to gluconeogenic operation in yeast.</em><br/>
Brauer <em>et al.</em>, Mol Biol Cell 2005: <em>Homeostatic adjustment and metabolic remodeling in glucose-limited yeast cultures.</em><br/>
Slavov <em>et al.</em>, Cell Rep 2014: <em>Constant growth rate can be supported by decreasing energy flux and increasing aerobic glycolysis.</em>
</font></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="lineheight" id="slide-19" style="background:;">
  <article data-timings="">
    <table><thead>
<tr>
<th>stress</th>
<th>translation vs. P-bodies</th>
</tr>
</thead><tbody>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/slavov14_selected_transcripts.png"  height="240"></td>
<td><img src="assets/img/slavov14_selected_proteins.png"  height="240"></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><img src="assets/img/zid14_transcripts.png"  height="240"></td>
<td><img src="assets/img/zid14_proteins.png"  height="240"></td>
</tr>
</tbody></table>

<p><font size=3> 
Slavov <em>et al.</em>, Cell Rep 2014: <em>Constant growth rate can be supported by decreasing energy flux and increasing aerobic glycolysis.</em></br>
Zid &amp; O&#39;Shea, Nature 2014: <em>Promoter sequences direct cytoplasmic localization and translation of mRNAs during starvation in yeast.</em>
</font></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-20" style="background:;">
  <hgroup>
    <h3>Stress &amp; Chromatin</h3>
  </hgroup>
  <article data-timings="">
    
<div style='float:left;width:35%;'>
  <p><img src="assets/img/amariei14_fig1b.png"  height="400"></p>

</div>
<div style='float:right;width:60%;'>
  <p><img src="assets/img/amariei14_fig3a.png"  height="100">
<img src="assets/img/amariei14_fig3c.png"  height="100">
<img src="assets/img/amariei14_fig3e-g.png"  height="300"></p>

</div>
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="class" id="id" style="background:;">
  <hgroup>
    <h3>Stress &amp; Chromatin</h3>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/nucl_nocetti16_movie_AD.gif" alt="Fritz">
<img src="assets/img/nucl_amariei14_movie_AD.gif" alt="Fritz"></p>

<p>-&gt; model chromatin remodeling, but ATP/ADP general, electron transport chain</p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="class" id="id" style="background:;">
  <hgroup>
    <h3>Pervasive Transcription</h3>
  </hgroup>
  <article data-timings="">
    <p>RNAseq data!</p>

<p>noncoding, antisense expression: only during low energy phase</p>

<p>upstream noncoding: detected by Karl, segmenTier, potentially new mechanism
after yeast WGD</p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>THE END</h2>
  </hgroup>
  <article data-timings="">
    <p>concepts and tools for cyano/bacteria, coilseq</p>

  </article>
  <!-- Presenter Notes -->
</slide>

    <slide class="backdrop"></slide>
  </slides>
  <div class="pagination pagination-small" id='io2012-ptoc' style="display:none;">
    <ul>
      <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=1 title='NA'>
         1
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=2 title='Growth &amp; Gene Expression in <em>E. coli</em>'>
         2
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=3 title='Growth &amp; Gene Expression in <em>E. coli</em>'>
         3
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=4 title='Varying Periods'>
         4
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=5 title='Varying Periods at Balanced Growth'>
         5
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=6 title='Varying Periods &amp; the Cell Cycle?'>
         6
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=7 title='Varying Periods ... Constant Gene Expression Profile'>
         7
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=8 title='Gene Expression : function'>
         8
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=9 title='Gene Expression : meta-transcriptome'>
         9
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=10 title='Gene Expression &amp; Growth Laws'>
         10
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=11 title='Gene Expression &amp; Growth Laws'>
         11
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=12 title='Gene Expression &amp; Growth Laws'>
         12
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=13 title='NA'>
         13
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=14 title='NA'>
         14
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=15 title='NA'>
         15
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=16 title='NA'>
         16
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=17 title='NA'>
         17
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=18 title='NA'>
         18
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=19 title='NA'>
         19
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=20 title='Stress &amp; Chromatin'>
         20
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=21 title='Stress &amp; Chromatin'>
         21
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=22 title='Pervasive Transcription'>
         22
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=23 title='THE END'>
         23
      </a>
    </li>
  </ul>
  </div>  <!--[if IE]>
    <script 
      src="http://ajax.googleapis.com/ajax/libs/chrome-frame/1/CFInstall.min.js">  
    </script>
    <script>CFInstall.check({mode: 'overlay'});</script>
  <![endif]-->
</body>
  <!-- Load Javascripts for Widgets -->
  
  <!-- MathJax: Fall back to local if CDN offline but local image fonts are not supported (saves >100MB) -->
  <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        inlineMath: [['$','$'], ['\\(','\\)']],
        processEscapes: true
      }
    });
  </script>
  <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
  <!-- <script src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  </script> -->
  <script>window.MathJax || document.write('<script type="text/x-mathjax-config">MathJax.Hub.Config({"HTML-CSS":{imageFont:null}});<\/script><script src="libraries/widgets/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><\/script>')
</script>
<!-- LOAD HIGHLIGHTER JS FILES -->
  <script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>
  <!-- DONE LOADING HIGHLIGHTER JS FILES -->
   
  </html>